packages/core/src/Input/Input.stories.tsx

```diff
@@ -21,27 +21,27 @@ const Template = (args) => <Input my={3} {...args} />
 
 export const InputComponent = Template.bind({})
 
-export const Colors = () => (
+export const Colors = (args) => (
   <Box width={400}>
-    <Input mb={3} id='input-colors-1' placeholder='No color' />
-    <Input mb={3} id='input-colors-2' color='primary' placeholder='Primary' />
-    <Input mb={3} id='input-colors-3' color='secondary' placeholder='Secondary' />
-    <Input mb={3} id='input-colors-4' color='warning' placeholder='Warning' />
-    <Input mb={3} id='input-colors-5' color='alert' placeholder='Alert' />
-    <Input mb={3} id='input-colors-6' color='caution' placeholder='Caution' />
+    <Input mb={3} id='input-colors-1' placeholder='No color' {...args} />
+    <Input mb={3} id='input-colors-2' color='primary' placeholder='Primary' {...args} />
+    <Input mb={3} id='input-colors-3' color='secondary' placeholder='Secondary' {...args} />
+    <Input mb={3} id='input-colors-4' color='warning' placeholder='Warning' {...args} />
+    <Input mb={3} id='input-colors-5' color='alert' placeholder='Alert' {...args} />
+    <Input mb={3} id='input-colors-6' color='caution' placeholder='Caution' {...args} />
```

Comment:
I know it's not entirely in the scope of your change, but I think we should stick to using args only with stories that are using a template. Otherwise, this might set an example for others to use args in regular stories, which isn't the [recommended pattern](https://storybook.js.org/docs/react/writing-stories/args).

Reply:
good to know. can you check if its good now @craigpalermo